src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -477,8 +554,16 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 	private actionItemProvider(action: Action): IActionViewItem {
 		// Check extensions to create ActionItem; otherwise, return undefined
 		// This is similar behavior that exists in MenuItemActionItem
+		let incomingLabel: string;
+
 		if (action instanceof MenuItemAction) {
-			return new LabeledMenuItemActionItem(action, this.keybindingService, this.contextMenuService, this.notificationService, 'notebook-button');
+
+			if (action.item.id.includes('jupyter.cmd') && this.previewFeaturesEnabled) {
+				incomingLabel = action.label;
```

Comment:
Remove the incomingLabel and just flip the two below statements 

```
action.tooltip = action.label;
action.label = '';
```

Reply:
Flipped (╯°□°)╯︵ ┻━┻